engine_bs/README.md

```diff
@@ -15,17 +15,53 @@ sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.
 opam init
 ```
 
-2. Cloning repositories
+2. BuckleScript
+
+Clone the BuckleScript fork that works with Sketch.sh. `bucklescript` root project folder
+should be next to `sketch-sh` root project folder in the directory structure.
 
 ```sh
-git clone https://github.com/BuckleScript/bucklescript
-git clone https://github.com/BuckleScript/bucklescript-playground
```

Comment:
@thangngoc89 another change was the removal of `bucklescript-playground` here. It's not needed right? as the "target playground" is sketch.

Reply:
Correct!